src/kv/deserialise.h

```diff
@@ -81,9 +81,11 @@ namespace kv
       auto search = changes.find(ccf::Tables::SIGNATURES);
       if (search != changes.end())
       {
-        // Transactions containing a signature must only contain
-        // a signature and must be verified
-        if (changes.size() > 1)
+        // Transactions containing a signature must only contain the signature
+        // and the serialised Merkle tree and must be verified
+        if (
```

Comment:
The serialised tree is not directly signed by the signature transaction but the historical queries module verifies that the root of the serialised tree matches the root that's signed in the signature transaction.